content/en/blog/_posts/2022-08-15-introducing-cosi.md

```diff
@@ -0,0 +1,91 @@
+* Portability - Vendor neutrality enabled through portability across Kubernetes Clusters and across Object Storage vendors
+
+_Portability across vendors is only possible when both vendors support a common datapath-API. Eg. it is possible to port from AWS S3 to Ceph, or AWS S3 to MinIO and back as they all use S3 API. In contrast, it is not possible to port from AWS S3 and Google Cloud’s GCS or vice versa._
+
+## Architecture
+
+COSI is made up of three components:
+
+* COSI Controller Manager
+* COSI Sidecar
+* COSI Driver
+
+The COSI Controller Manager acts as the main controller that processes changes to COSI API objects. It is responsible for fielding requests for bucket creation, updates, deletion and access management. One instance of the controller manager is required per kubernetes cluster. Only one is needed even if multiple object storage providers are used in the cluster. 
+
+The COSI Sidecar acts as a translator between COSI API requests and vendor specific COSI Drivers. This component uses a standardized gRPC protocol that vendor drivers are expected to satisfy. 
+
+The COSI Driver is the vendor specific component that receives requests from the sidecar and calls the appropriate vendor APIs to create buckets, manage their lifecycle and manage access to them. 
+
+## API
+
+The COSI API is centered around buckets, since they are the unit of abstraction for provisioning object storage. We define three kubernetes APIs aimed at managing them
+
+* Bucket
+* BucketClass
+* BucketClaim
+
+In addition, two more APIs for managing access to buckets are also defined:
+
+* BucketAccess
+* BucketAccessClass
+
+In a nutshell, Bucket and BucketClaim can be considered to be similar to PersistentVolume and PersistentVolumeClaim respectively. The BucketClass’s counterpart in the file/block device world is StorageClass. 
+
+Since Object Storage is always authenticated, and over the network, access credentials are required to access buckets. The two APIs, namely, BucketAccess and BucketAccessClass are used to denote access credentials and policies for authentication. More info about these APIs can be found in the official COSI proposal - https://github.com/kubernetes/enhancements/pull/2813
+
+## Self-Service
+
+Other than providing kubernetes-API driven bucket management, COSI also aims to empower DevOps personnel to provision and manage buckets on their own, without admin intervention. This, further enabling dev teams to realize faster turn-around times and faster time-to-market. 
+
+COSI achieves this by dividing bucket provisioning steps among two different stakeholders, namely the administrator (admin), and the operator (devops). The administrator will be responsible for setting broad policies and limits on how buckets are provisioned, and how access is obtained for them. The operator will be free to create and utilize buckets within the limits set by the admin. 
+
```

Comment:
Can you use "devops" instead of "operator" because "operator" has other meanings as well

Reply:
Our official term is “cluster operator”, BTW.

Reply:
Got it